### ISSUE TITLE
Not showing Sidecar options in System Preferences

### ISSUE THREAD

**Issue by cubcrafts** [ben-z/free-sidecar#33]
Please fill in the template below to speed up the review process:

MacBook Pro (Retina, 15-inch, Mid 2015)
macOS Catalina 10.15.1

iPad 9.7" 5th Gen (Wi-Fi/Cellular) - iPad6,12
iOS 13.3.1

I ran through all steps of the installation as described, I had to update xcode before I could run the commands in step 7 & 8 but afterwards they ran fine with no errors. After completing everything, Sidecar is not showing up in system preferences. I have my iPad connected via USB A - Lighting cable and I tried to sign out and back in of iCloud in the iPad, both devices are also on the same WiFi network with no VPN active (incase that make a difference).

Any help is appreciated

![Screenshot 2020-01-07 at 12 24 53](https://user-images.githubusercontent.com/52754382/71895362-cb832f80-3148-11ea-8146-0580cde786dc.png)

**Comment by ben-z** [comment]
Which MacBook version did you patch? I think MacBook Pro 2015 is `MacBookPro12,1`.

**Comment by cubcrafts** [comment]
Yes it was MacbookPro12,1
<img width="863" alt="Screenshot 2020-01-07 at 17 33 43" src="https://user-images.githubusercontent.com/52754382/71915673-267e4c00-3174-11ea-8697-4e3c0698ae6b.png">
<img width="863" alt="Screenshot 2020-01-07 at 17 33 50" src="https://user-images.githubusercontent.com/52754382/71915675-267e4c00-3174-11ea-8093-dd95fc0f45b1.png">

**Comment by f-fritz** [comment]
While the 13“ MacBook Pro from 2015 has the 12,X identifier, the 15“ MacBook Pro from 2015 has  11,4 (base model) or 11,5 (higher end models).
You should look that up using the System Profiler tool (or About this Mac > System Report).

**Comment by ben-z** [comment]
Omg that’s probably it. For some reason I assumed OP had a 13-inch ‘15 MBP. Thanks @f-fritz for pointing it out!

**Comment by durquharttax** [comment]
My apologies, I should have mentioned it was a 15 inch. I have now installed the 11,4 and SideCar is now showing correctly in System Preferences however I cannot connect to my iPad wired or wirelessly.

**Comment by cubcrafts** [comment]
My apologies, I should have mentioned it was a 15 inch. I have now installed the 11,4 and SideCar is now showing correctly in System Preferences however I cannot connect to my iPad wired or wirelessly. 

Edit: I tried the other USB port on my laptop and its working fine now! Didnt expect my USB port to be the issue but ah well. Still not working wirelessly however

Thanks for this amazing software Ben!
